rpm/cray/csm/sle-15sp2/index.yaml

```diff
@@ -21,37 +21,30 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-http://car.dev.cray.com/artifactory/csm/CSM/sle15_sp2_ncn/noarch/release/csm-1.0/:
-  rpms:
-    - hpe-csm-goss-package-0.3.13-20210615152800_aae8d77.noarch
-    - hpe-csm-yq-package-3.4.1-20210615153837_40f15a6.noarch
-http://car.dev.cray.com/artifactory/csm/CSM/sle15_sp2_ncn/x86_64/release/csm-1.0/:
-  rpms:
-    - loftsman-1.1.0-20210511145236_2da0507.x86_64
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/:
   rpms:
-    - cray-site-init-1.16.3-1.x86_64
+    - cray-site-init-1.16.7-1.x86_64
     - dracut-metal-dmk8s-1.5.2-1.noarch
     - dracut-metal-luksetcd-1.5.4-1.noarch
-    - dracut-metal-mdsquash-1.6.4-1.noarch
+    - dracut-metal-mdsquash-1.9.3-1.noarch
     - metal-basecamp-1.1.9-1.x86_64
     - metal-net-scripts-0.0.2-1.noarch
-    - pit-init-1.2.12-1.noarch
+    - pit-init-1.2.14-1.noarch
     - pit-nexus-1.1.0-1.1.x86_64
     - ilorest-3.2.3-1.x86_64
     - cf-ca-cert-config-framework-2.3.38-1.x86_64
     - cfs-state-reporter-1.7.50-1.x86_64
     - cfs-trust-1.3.94-1.x86_64
-    - cray-cmstools-crayctldeploy-1.2.116-1.x86_64
+    - cray-cmstools-crayctldeploy-1.3.3-1.x86_64
     - cray-switchboard-1.2.3-1.x86_64
     - cray-uai-util-2.1.0-1.x86_64
-    - craycli-0.48.0-1.x86_64
+    - craycli-0.46.0-1.x86_64
```

Comment:
craycli-0.48.0 should be preserved here.

Reply:
Done, good catch